app/models/setting.rb

```diff
@@ -0,0 +1,11 @@
+class Setting < ApplicationRecord
+  belongs_to :user
+
+  enum locale: [:en, :pl]
+
+  def deafault_locale
```

Comment:
This method can be refactor. Since we always have setting. It is created during user creation and we updated database with after party task you don't need this condition.